migrations/20210101_initial.sql

```diff
@@ -1,89 +1,68 @@
-CREATE TABLE IF NOT EXISTS servers(
+CREATE TABLE servers (
   host TEXT NOT NULL,
   port TEXT NOT NULL,
   key_hash BLOB,
   PRIMARY KEY (host, port)
 ) WITHOUT ROWID;
 
-CREATE TABLE IF NOT EXISTS rcv_queues(
+CREATE TABLE connections (
+  conn_alias BLOB NOT NULL PRIMARY KEY,
+  conn_mode TEXT NOT NULL,
+  last_internal_msg_id INTEGER NOT NULL DEFAULT 0,
+  last_internal_rcv_msg_id INTEGER NOT NULL DEFAULT 0,
+  last_internal_snd_msg_id INTEGER NOT NULL DEFAULT 0,
+  last_external_snd_msg_id INTEGER NOT NULL DEFAULT 0,
+  last_rcv_msg_hash BLOB NOT NULL DEFAULT x'',
+  last_snd_msg_hash BLOB NOT NULL DEFAULT x''
+) WITHOUT ROWID;
+
+CREATE TABLE rcv_queues (
   host TEXT NOT NULL,
   port TEXT NOT NULL,
   rcv_id BLOB NOT NULL,
-  conn_alias BLOB NOT NULL,
+  conn_alias BLOB NOT NULL REFERENCES connections ON DELETE CASCADE,
   rcv_private_key BLOB NOT NULL,
+  rcv_dh_secret BLOB NOT NULL,
   e2e_priv_key BLOB NOT NULL,
   e2e_snd_pub_key BLOB,
   e2e_dh_secret BLOB,
   snd_id BLOB NOT NULL,
   snd_key BLOB,
   status TEXT NOT NULL,
   PRIMARY KEY (host, port, rcv_id),
-  FOREIGN KEY (host, port) REFERENCES servers (host, port),
-  FOREIGN KEY (conn_alias)
-    REFERENCES connections (conn_alias)
-    ON DELETE CASCADE
-    DEFERRABLE INITIALLY DEFERRED,
+  FOREIGN KEY (host, port) REFERENCES servers,
```

Comment:
not sure - I don't remember if we discussed it before
```suggestion
  FOREIGN KEY (host, port) REFERENCES servers ON DELETE CASCADE
```